test/fixtures/check-direct-import-transitive-dependency.txt

```diff
@@ -5,4 +5,6 @@ These dependencies are unused. To fix this warning, remove the following package
 - effect[0m
 [31m[error] [0mSome of your project files import modules from packages that are not in the direct dependencies of your project.
 To fix this error add the following packages to the list of dependencies in your config:
-- prelude[0m
+- prelude
+You may install those packages by running the following command:
```

Comment:
Do you think it would be worth changing the wording here slightly to be something like "You may add these dependencies by running the following command:"?

The reason I ask is that these dependencies are already installed at this point (because they are transitive dependencies and those all get installed, and the build would have failed if they were not), so really all you need to do is list them as a dependency.